### PR TITLE
Documentation for path based scanning and upc optimisation

### PR DESCRIPTION
**Pull request by juhi037**

### CODE REVIEW

File: 02- Running ADO Scanner from command line/Readme.md [hunk 1]
```diff
@@ -122,6 +124,22 @@ The Get-AzSKADOSecurityStatus command now supports checkpointing via a "-UsePart
 Get-AzSKADOSecurityStatus-OrganizationName "<OrganizationName>" -ScanAllResources -UsePartialCommits
 ```
 
+#### Speed up checkpointed scans with "-DoNotRefetchResources" switch
+The "-UsePartialCommits" switch also supports an optional switch: "-DoNotRefetchResources" in SDL mode. When this switch is used, resources are not re-fetched during the continuation of the checkpointed scan (i.e., when the "-upc" switch is used). This efficiently speeds up scans of subsequent batches after the initial one. Currently the resources supported with the switch are: Release, Agent Pool, Organization and Project. 
+
+```PowerShell
+Get-AzSKADOSecurityStatus-OrganizationName "<OrganizationName>" -ProjectName "<ProjectName>" -ReleaseNames * -ResourceTypeName -Release -UsePartialCommits -DoNotRefetchResources
```
Comment (Garima-msft) [inline 02- Running ADO Scanner from command line/Readme.md:131]: add space between "Get-AzSKADOSecurityStatus"  & "-OrganizationName" in all commands

File: 02- Running ADO Scanner from command line/Readme.md [hunk 2]
```diff
@@ -122,6 +124,22 @@ The Get-AzSKADOSecurityStatus command now supports checkpointing via a "-UsePart
 Get-AzSKADOSecurityStatus-OrganizationName "<OrganizationName>" -ScanAllResources -UsePartialCommits
 ```
 
+#### Speed up checkpointed scans with "-DoNotRefetchResources" switch
+The "-UsePartialCommits" switch also supports an optional switch: "-DoNotRefetchResources" in SDL mode. When this switch is used, resources are not re-fetched during the continuation of the checkpointed scan (i.e., when the "-upc" switch is used). This efficiently speeds up scans of subsequent batches after the initial one. Currently the resources supported with the switch are: Release, Agent Pool, Organization and Project.
```
Comment (SaikumarRagala) [inline 02- Running ADO Scanner from command line/Readme.md:128]: remove ":"

File: 02- Running ADO Scanner from command line/Readme.md [hunk 3]
```diff
@@ -122,6 +124,22 @@ The Get-AzSKADOSecurityStatus command now supports checkpointing via a "-UsePart
...
+The "-UsePartialCommits" switch also supports an optional switch: "-DoNotRefetchResources" in SDL mode. When this switch is used, resources are not re-fetched during the continuation of the checkpointed scan (i.e., when the "-upc" switch is used). This efficiently speeds up scans of subsequent batches after the initial one. Currently the resources supported with the switch are: Release, Agent Pool, Organization and Project. 
+
+```PowerShell
+Get-AzSKADOSecurityStatus -OrganizationName "<OrganizationName>" -ProjectName "<ProjectName>" -ReleaseNames * -ResourceTypeName -Release -UsePartialCommits -DoNotRefetchResources
+```
+
+----------------------------------------------
+
+### Execute path based scanning for builds and releases
+The Get-AzSKADOSecurityStatus command supports path based scanning by scanning build and release configs constrained to specific paths using two switches: "-BuildsFolderPath" and "-ReleasesFolderPath"
```
Comment (SaikumarRagala) [inline 02- Running ADO Scanner from command line/Readme.md:137]: can we specify what path means here?

**Review by v-himkam — APPROVED** [review]

**Review by Garima-msft — DISMISSED** [review]

File: 02- Running ADO Scanner from command line/Readme.md [hunk 4]
```diff
@@ -122,6 +124,29 @@ The Get-AzSKADOSecurityStatus command now supports checkpointing via a "-UsePart
 Get-AzSKADOSecurityStatus-OrganizationName "<OrganizationName>" -ScanAllResources -UsePartialCommits
 ```
 
+#### Speed up checkpointed scans with "-DoNotRefetchResources" switch
+The "-UsePartialCommits" switch also supports an optional switch: "-DoNotRefetchResources" in SDL mode. When this switch is used, resources are not re-fetched during the continuation of the checkpointed scan (i.e., when the "-upc" switch is used). This efficiently speeds up scans of subsequent batches after the initial one. Currently the resources supported with the switch are Release, Agent Pool, Organization and Project. 
+
+```PowerShell
+Get-AzSKADOSecurityStatus -OrganizationName "<OrganizationName>" -ProjectName "<ProjectName>" -ReleaseNames * -ResourceTypeName -Release -UsePartialCommits -DoNotRefetchResources
```
Comment (v-himkam) [inline 02- Running ADO Scanner from command line/Readme.md:131]: Remove "-" from release after -resourceTypeName

**Review by v-himkam — APPROVED** [review]

**Review by SaikumarRagala — APPROVED** [review]

**Pull request merged**